tests/test_schema/test_schema_uptodate.py

```diff
@@ -13,7 +13,7 @@ def test_schema_uptodate():
     To update the local schema, run:
     `./tools/update_schema`
     """
-    with open("schema/definitions/0.8.0/schema/fmu_results.json") as f:
+    with open("schemas/0.8.0/fmu_results.json") as f:
```

Comment:
While in here, should we take the version from the `FmuResultsSchema.VERSION` maybe?

Reply:
Yes, good point. I'm touching a lot of this code in a PR I'm working on now so I'm going skip it for this one and include in the next one